examples/custom_docker_image/Dockerfile

```diff
@@ -0,0 +1,12 @@
+##  Could be any base image, only build dependencies are git and pip
+FROM python:3.8-slim
+
+ENV PROJECT_NAME='commands-base-example'
+
+RUN apt-get update && apt-get install -y git && \
+    pip install git+https://github.com/fivestars-os/commands-base@2.0.1 && \
+    apt-get clean
```

Comment:
`apt-get clean` is [not necessary on ubuntu- or debian-based images](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). However, it's still useful to do `rm -rf /var/lib/apt/lists/*` instead

Reply:
neat!